sda/internal/database/db_functions.go

```diff
@@ -229,20 +229,20 @@ func (dbs *SDAdb) getHeader(fileID string) ([]byte, error) {
 }
 
 // MarkCompleted marks the file as "COMPLETED"
-func (dbs *SDAdb) MarkCompleted(file FileInfo, fileID, corrID string) error {
+func (dbs *SDAdb) SetVerified(file FileInfo, fileID, corrID string) error {
 	var (
 		err   error
 		count int
 	)
 
 	for count == 0 || (err != nil && count < RetryTimes) {
-		err = dbs.markCompleted(file, fileID, corrID)
```

Comment:
I am experiencing something funky, how come I can see this change here in the PR but in the branch `sda_fix_cancel` it seems still as `markCompleted`?

Reply:
Is your local branch up to date?

Reply:
I actually had checked it through the web. anyway, it looks right now